token/program-2022/src/extension/confidential_transfer/instruction.rs

```diff
@@ -510,44 +543,71 @@ pub struct WithdrawWithheldTokensFromAccountsData {
 }
 
 /// Create a `InitializeMint` instruction
+#[cfg(not(target_os = "solana"))]
 pub fn initialize_mint(
     token_program_id: &Pubkey,
     mint: &Pubkey,
-    ct_mint: &ConfidentialTransferMint,
+    authority: Option<&Pubkey>,
+    auto_approve_new_accounts: bool,
+    auditor_encryption_pubkey: &EncryptionPubkey,
+    withdraw_withheld_authority_encryption_pubkey: &EncryptionPubkey,
 ) -> Result<Instruction, ProgramError> {
     check_program_account(token_program_id)?;
     let accounts = vec![AccountMeta::new(*mint, false)];
+
+    // TODO: use `OptionalNonZeroPubkey`
```

Comment:
It's too late for this, but we should avoid new `TODO`s in the code, since they typically never get done. Can you add an issue tracking this?

Reply:
Got it. The issue was created #3941.